docs/en/Configuring-Alluxio-with-secure-HDFS.md

```diff
@@ -62,6 +62,13 @@ Then edit `alluxio-site.properties` file to set the under storage address to the
 Copy secure HDFS conf xml files (`core-site.xml`, `hdfs-site.xml`, `mapred-site.xml`, `yarn-site.xml`) to
 `${ALLUXIO_HOME}/conf/`
 
+Optionally, you can set jvm-level system properties for customized Kerberos configurations:
```

Comment:
When you want customized Kerberos configs, *different from* the default config on your machine, you should set those two properties explicitly. Otherwise you do not need to set them because the default Kerberos config on the machine got picked up.